Source/WebInspectorUI/UserInterface/Controllers/CSSManager.js

```diff
@@ -306,6 +307,19 @@ WI.CSSManager = class CSSManager extends WI.Object
         return this._defaultAppearance;
     }
 
+    get defaultUserPreferences()
+    {
+        return this._defaultUserPreferences;
+    }
+
+    set defaultUserPreferences(userPreferences)
+    {
+        for (let userPreference of userPreferences)
+            this.defaultUserPreferences.set(userPreference.name, userPreference.value)
+    }
+
+    get overridenUserPreferences() { return this._overridenUserPreferences; }
```

Comment:
Style: single line `get` should be at the top of the `// Public` section (after any other single line `get`)

Reply:
Done.